sample-app/src/utils/searchhandler.ts

```diff
@@ -0,0 +1,47 @@
+import { AnswersActions } from "../../../lib";
+
+
+const defaultGeolocationOptions: PositionOptions = {
+  enableHighAccuracy: false,
+  timeout: 6000,
+  maximumAge: 300000,
+};
+
+/**
+ * Provide utility functions related to vertical and universal search execution.
+ * This includes any potential data check or setup related to a query before
+ * performing a search, such as fetching searchIntents using autocomplete requests,
+ * and retrieving user's location using nagivator.geolocation API.
+ */
+export default class SearchHandler {
+  static async executeSearch(answersActions: AnswersActions, isVertical: boolean) {
```

Comment:
Are there advantages to putting static methods on a class, as opposed to just having separate methods? Maybe we could use a typescript name space instead? afaik static methods are not treeshakeable

Reply:
The class is just a way to group these functions into a unit related to search. With in being a class, I initially consider having a constructor to pass in answersAction and isVertical but then not all functions really not that (getUserLocation). I made them statics since it's not necessary to instantiate new SearchHandler. I don't have a strong preference between utility classes vs utility functions